dist/init/linux-upstart/caddy.conf

```diff
@@ -1,18 +1,25 @@
-description "Caddy startup script"
+description "Caddy HTTP/2 web server"
 author "Mathias Beke"
 
 start on runlevel [2345]
 stop on runlevel [016]
 
+console log
 
 setuid www-data
 setgid www-data
 
 respawn
 respawn limit 10 5
 
-limit nofile 4096 4096
+# Letsencrypt-issued certificates will be written to this directory.
+env HOME=/etc/ssl/caddy
```

Comment:
Just FYI, in the future, the .caddy folder may store more than just certificates, so it won't always be SSL-related. Also, Caddy will make a .caddy folder inside $HOME, so in this case it'd be `/etc/ssl/caddy/.caddy` - I'd just use an actual user HOME directory personally.